Added 'screen' method (#400)
rudderlabs/rudderstack-docs: stream-sources/rudderstack-sdk-integration-guides/rudderstack-unity-sdk.mdx

rudderClient.Identify("some_user_id", traits, identifyMessage);
```

## Screen

> The `screen` method lets you record your website's page views via users on their mobile screen. It also includes any additional relevant information about the screen.

Reviewer: ```suggestion
The `screen` call lets you record whenever a user views their mobile screen with any additional relevant information about the viewed screen.
```
